hw/ip/csrng/data/csrng_sec_cm_testplan.hjson

```diff
@@ -24,8 +24,8 @@
 {
   testpoints: [
     {
-      // TODO: Unlike the REGWEN register, the CTRL and ERR_CODE_TEST registers are excluded from some automated CSRs tests (CsrExclWrite).
-      // We need to ensure write-read checks are happening or alternatively test point 2) explicitly, e.g., with a directed test.
+      // SinceC TRL and ERR_CODE_TEST registers are excluded from some automated CSRs tests (CsrExclWrite), test points
+      // 1) and 2) are verified using a directed test.
```

Comment:
Note that there is a typo here: SinceC TRL -> Since CTRL

Reply:
Also, please switch to 1 line per sentence.

Reply:
corrected now

Reply:
Thanks @vrozic !